openreview/conference/templates/tabsConferenceDecisionsWebfield.js

```diff
@@ -138,35 +149,52 @@ function createConsoleLinks(allGroups) {
   });
 }
 
-function groupNotesByDecision(notes, decisionNotes, withdrawnNotes, deskRejectedNotes) {
-  // Categorize notes into buckets defined by DECISION_HEADING_MAP
-  var notesDict = _.keyBy(notes, 'id');
+function renderNotesbyDecision(submissionCount, submissions, venueId) {
+    
+  var container = '#' + getElementId(DECISION_HEADING_MAP[venueId])
 
-  var papersByDecision = {};
-  for (var decision in DECISION_HEADING_MAP) {
-    papersByDecision[getElementId(DECISION_HEADING_MAP[decision])] = [];
-  }
-
-  decisionNotes.forEach(function(d) {
-    var tabName = DECISION_HEADING_MAP[d.content.decision];
-    if (tabName) {
-      var decisionKey = getElementId(tabName);
-      if (notesDict[d.forum] && papersByDecision[decisionKey]) {
-        papersByDecision[decisionKey].push(notesDict[d.forum]);
-      }
-    }
-
-  });
+  $(container).empty();
+  if (submissionCount) {
+    var SearchResultsListOptions = _.assign({}, paperDisplayOptions, {
```

Comment:
I think this should be `searchResultsListOptions`, right?

Reply:
Fixed.